integration_tests/test_doc.py

```
@@ -0,0 +1,256 @@
+    receipt = await PRECOMPILE.fns.grantRole(
+        REGISTRY_ID, checksum, editor.address, Role.EDITOR
+    ).transact(w3, admin, to=DOCUMENT)
+    assert receipt.status == 1, "Setup grantRole failed"
+
+    tx = PRECOMPILE.fns.revokeRole(REGISTRY_ID, checksum, editor.address)
+
+    if should_succeed:
+        receipt = await tx.transact(w3, sender, to=DOCUMENT)
+        assert receipt.status == 1, "RevokeRole transaction failed"
+    else:
+        with pytest.raises(Exception):
```

Ruff (0.14.8)

Do not assert blind exception: `Exception`

(B017)